rslib/src/scheduler/fsrs/weights.rs

```diff
@@ -299,10 +336,14 @@ pub(crate) mod tests {
 
     const NEXT_DAY_AT: TimestampSecs = TimestampSecs(86400 * 100);
 
+    pub(crate) fn days_ago_ms(days_ago: i64) -> TimestampMillis {
```

Comment:
If this and the method below don't need to be used in other files, the pub(crate) is unnecessary.